### PR TITLE
add prefix check for adding pv to cache

### PR DESCRIPTION
**Pull request by yuyulei**
If node recycled for some reasons(e.g. add cpu/mem/ip capacity, hardware error), then the node rejoined the cluster, node name not changes, but uid changes.

The problem is that some pvs is created before node recycling, now they not match node uid.

```
pv.kubernetes.io/provisioned-by: local-volume-provisioner-<node name>-<old uid>
```

So provisiner will not add these pvs to cache. 

Now I want use `UserConfig.UseNodeNameOnly` to ignore node uid info when create pv. Further, I also hope the provisioner can add old pvs to cache. At last, I make the pr.

### CODE REVIEW

**Comment by yuyulei** [comment]
@cofyc

File: pkg/populator/populator.go [hunk 1]
```diff
@@ -72,7 +73,7 @@ func (p *Populator) handlePVUpdate(pv *v1.PersistentVolume) {
 			if !found {
 				return
 			}
-			if provisioner == p.Name {
+			if provisioner == p.Name || strings.HasPrefix(provisioner, p.Name) {
```
Comment (yuyulei) [inline pkg/populator/populator.go]: may I  use "strings.HasPrefix(provisioner, p.Name) " to replace "provisioner == p.Name "?

**Comment by yuyulei** [comment]
@cofyc

**Comment by cofyc** [comment]
after changing `UseNodeNameOnly` to `true` from `false`, how about writing a script to migrate existing PVs to the new provisioner

**Pull request closed without merging**